project/MoneyBuild.scala

```diff
@@ -285,6 +303,14 @@ object MoneyBuild extends Build {
     val codahaleVersion = "3.0.2"
     val apacheHttpClientVersion = "4.3.5"
 
+    val akkaV = "2.4.14"
```

Comment:
are we sure about these akka and akka-http versions?  Not sure if they are the current

Reply:
Current would be `akkaV = 2.5.11` and `akka-http = 10.1.0` but we aren't running that as of right now on PAPI. Not sure what the effect would be of having money depend on the latest version. I can take a look.